solar_consumer/data/fetch_gb_data.py

```diff
@@ -131,9 +129,16 @@ def fetch_gb_data_historic(regime: str) -> pd.DataFrame:
             f"Got {len(gsp_yield_df)} gsp yield for gsp id {gsp_id} before filtering"
         )
 
-        # TODO if did not find any values,
         # https://github.com/openclimatefix/solar-consumer/issues/104
-        # Make nighttime zeros
+        # Make nighttime zeros using pvlib solar elevation (needs lat/lon for this GSP)
+        gsp_yield_df = make_night_time_zeros(
```

Comment:
could you add an if statement like [this](https://github.com/openclimatefix/pvlive-consumer/blob/main/pvliveconsumer/nightime.py#L41)